### ISSUE TITLE
5.4 No hint path defined for [mail]

### ISSUE THREAD

**Issue by nebroht** [jrean/laravel-user-verification#102]
I always get the error "No hint path defined for [mail]" if i customize the email.blade.php with the new markdown feature of Laravel 5.4.

**Comment by jrean** [comment]
@nebroht can you please share:
- [x] the error trace
- [x] the customized `email.blade.php` file

Thxs

**Comment by jrean** [comment]
Ok, I replicated.

```
ErrorException in FileViewFinder.php line 112:
No hint path defined for [mail].
```

Markdown is only available for "Mailable". This package is still currently relying on the `Mailer`
https://laravel.com/api/5.4/Illuminate/Mail/Mailer.html.

It will be implemented in the 4.1 upcoming release.

**Comment by jrean** [comment]
https://github.com/jrean/laravel-user-verification/releases/tag/v4.1.0
